notebooks/02-listas_y_for_loops.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Listas y for loops\n",
```

Comment:
<p>¿Qué te parece introducir las tuplas más adelante? Quizás al final del notebook, como para no marearlos. Mi idea sería que se familiaricen con las listas y que se acostumbren a definirlas con <code>[ ]</code>  y al final mostrar que existen las tuplas.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoLatinas/Intro-to-python/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>